### PR TITLE
Update bindService to use getPreferredVersion

### PR DESCRIPTION
**Pull request by benjaminapetersen**

### CODE REVIEW

File: dist/origin-web-common-services.js [hunk 1]
```diff
@@ -1140,7 +1140,8 @@ angular.module("openshiftCommonServices")
     // The secret key this service uses for the parameters JSON blob when binding.
     var PARAMETERS_SECRET_KEY = 'parameters';
 
-    var bindingResource = APIService.getPreferredVersion('servicebindings');
+    var serviceBindingVersion = APIService.getPreferredVersion('servicebindings');
```
Comment (benjaminapetersen) [inline dist/origin-web-common-services.js]: just changed the name here to standardize w/all the others

File: src/services/bindService.js [hunk 2]
```diff
@@ -11,7 +11,8 @@ angular.module("openshiftCommonServices")
     // The secret key this service uses for the parameters JSON blob when binding.
     var PARAMETERS_SECRET_KEY = 'parameters';
 
-    var bindingResource = APIService.getPreferredVersion('servicebindings');
+    var serviceBindingVersion = APIService.getPreferredVersion('servicebindings');
```
Comment (spadgett) [inline src/services/bindService.js]: serviceBindingsVersion (plural) to be consistent

Comment (benjaminapetersen) [reply src/services/bindService.js]: Updated

**Comment by benjaminapetersen** [comment]
rebased

**Comment by spadgett** [comment]
OK to merge when travis passes

**Pull request merged**